stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -57,6 +57,9 @@ spec:
               mountPath: /etc/rabbitmq
             - name: {{ .Values.persistentVolume.name }}
               mountPath: /var/lib/rabbitmq
+{{- if .Values.extraInitContainers }}
+{{ tpl .Values.extraInitContainers . | indent 8 }}
```

Comment:
Why not `{{ tpl (toYaml .Values.extraInitContainers) . | indent 8 }}` to avoid extraInitContainers being a string?

Reply:
You have a good point, I made the required changes